# HG changeset patch
# User Hans
# Date 1486059825 0
#      Thu Feb 02 18:23:45 2017 +0000
# Branch trunk
# Node ID ca505206cc66108cc22fd902dd21b06ef8228335
# Parent  f8d69379fe552204e440feb02b97449ffcc6a6b2
Fix: Center extra viewport.

diff -r f8d69379fe55 -r ca505206cc66 src/viewport_gui.cpp
--- a/src/viewport_gui.cpp	Mon Jan 16 18:45:36 2017 +0000
+++ b/src/viewport_gui.cpp	Thu Feb 02 18:23:45 2017 +0000
@@ -71,7 +71,7 @@
 			pt.x = w->viewport->scrollpos_x + w->viewport->virtual_width / 2;
 			pt.y = w->viewport->scrollpos_y + w->viewport->virtual_height / 2;
 		} else {
-			pt = RemapCoords(TileX(tile) * TILE_SIZE + TILE_SIZE / 2, TileY(tile) * TILE_SIZE + TILE_SIZE / 2, TileHeight(tile));
+			pt = RemapCoords(TileX(tile) * TILE_SIZE + TILE_SIZE / 2, TileY(tile) * TILE_SIZE + TILE_SIZE / 2, TileHeight(tile) * TILE_HEIGHT);
 		}
 
 		this->viewport->scrollpos_x = pt.x - this->viewport->virtual_width / 2;
